### PR TITLE
Add Afterburner support to Local Server

### PR DESCRIPTION
**Pull request by joehoyle**

### CODE REVIEW

File: docs/using-afterburner.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Using Afterburner
+
+If your project uses Afterburner in Altis Cloud, you can enabled it in Altis Local Service by setting the `altis.modules.local-server.afterburner` value to `true`:
```
Comment (mikelittle) [inline docs/using-afterburner.md:3]: "enable" not "enabled"

**Review by mikelittle — CHANGES_REQUESTED** [review]
Looks good. Just a typo in the docs. And a lint issue about quotes.

File: docs/using-afterburner.md (hunk 1 above)
Comment (joehoyle) [reply docs/using-afterburner.md]: Ah good catch! Fixed now

**Review by mikelittle — APPROVED** [review]
There's still a linter issue about double quotes vs single quotes, but otherwise, looks good to me

**Review by jerico — APPROVED** [review]

**Pull request merged**